ecolex/legislation.py

```diff
@@ -129,25 +119,39 @@ def harvest_file(upfile):
             key = v['en2'].lower()
             all_languages[key] = v
 
+    resp = requests.get(settings.KEYWORDS_URL)
+    bs = BeautifulSoup(resp.content.decode(), 'xml')
+    keywords = bs.findAll('dictionary_term')
+    keyword_dict = {keyword.Code.string:[keyword.Name_en_US.string,
+                    keyword.Name_fr_FR.string, keyword.Name_es_ES.string]
+                    for keyword in keywords}
```

Comment:
Do you also use the list `keywords`? If not, you can create the dictionary under the same name:
```keywords = {keyword.Code.string: keyword for keyword in bs.findAll('dictionary_term')}```

Also, having the name values in a list (`[keyword.Name_en_US.string, keyword.Name_fr_FR.string, keyword.Name_es_ES.string]`) decreases the code readability: `keyword_dict[field_value][0]` means you should remember which value was on each position. That is why I suggested keeping the `keyword` object as a value in the dict.